versioned_docs/version-8.7/components/modeler/web-modeler/process-application-pipeline.md

```diff
@@ -66,7 +68,7 @@ Play is being rebuilt and progressively rolled out to more users. See [Play limi
 
 ## Review
 
-After validation is complete, you can create a new version of your process application and request a review:
+After validation is complete, you can [create a new version of your process application](/docs/components/modeler/web-modeler/process-application-versioning.md#create-a-bulk-version) and request a review:
```

Comment:
I tend to remove /docs/ from links, unless you are trying to link specifically to whatever the "next" version would be with these docs -- if you remove the /docs/ part, it will keep the user in whatever version of docs they are currently in.